tests/x11/oomath.pm

```diff
@@ -33,18 +33,20 @@ sub run {
     my $retries = 7;
     for (1 .. $retries) {
         type_string_slow "E %PHI = H %PHI\nnewline\n1 = 1";
+        wait_still_screen(2);
```

Comment:
this looks like you want to fix a different problem. I don't see why this should be necessary when afterwards we have a check screen anyway. In case you think this is still needed then replace the wait_still_screen+check_screen with a proper assert_screen call

Reply:
changed